napari/utils/events/event.py

```diff
@@ -517,7 +522,23 @@ def _normalize_cb(self, callback) -> Union[CallbackRef, Callback]:
         if isinstance(callback, tuple) and not isinstance(
             callback[0], weakref.ref
         ):
-            callback = (weakref.ref(callback[0]),) + callback[1:]
+            callback = (weakref.ref(callback[0]), callback[1])
+
+        if isinstance(callback, tuple):
+            callback_fun = getattr(callback[0](), callback[1])
+            signature = inspect.signature(callback_fun)
+            allow_event = any(
+                map(
+                    lambda x: x.kind
+                    in [
+                        inspect.Parameter.POSITIONAL_ONLY,
+                        inspect.Parameter.POSITIONAL_OR_KEYWORD,
+                        inspect.Parameter.VAR_POSITIONAL,
+                    ],
+                    signature.parameters.values(),
+                )
+            )
```

Comment:
this may be changed to 
```python
			allow_event = any(
                map(
                    lambda x: n.name == "event" and x.kind
                    in [
                        inspect.Parameter.POSITIONAL_ONLY,
                        inspect.Parameter.POSITIONAL_OR_KEYWORD,
                    ],
                    signature.parameters.values(),
                )
            )
```

to pass only event when the function has event field in signature. This should allow to pass event as a keyword.